_sass/helix/_icon.scss

```diff
@@ -22,6 +22,7 @@
   /* Better Font Rendering =========== */
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
```

Comment:
hey!  it looks like there's a typo in your stylesheet here, and it's made the icons look all janky.